k8s/federal/sync.yaml

```diff
@@ -123,7 +99,7 @@ spec:
   interval: 5m
   update:
     strategy: Setters
-    path: ./k8s/federal/rshiny-dashboard
```

Comment:
@simardeep1792  Are these changes expected or a merging fluke on this branch? Please double check the k8s manifests in particular while reviewing!

Reply:
Meant to multi-line comment that, there's a bunch of changes in `k8s/federal/sync.yaml` that I'm unsure of here.